### PR TITLE
Renumber the articles when the order changes

### PR DESCRIPTION
**Pull request by lagartoverde**
In the end this was easier than I expected

### CODE REVIEW

**Comment by lagartoverde** [comment]
Important, this assumes everything the user changes is inside the prov:value of the article, not sure if this is like this or even if we would want to be like this in the future

File: addon/commands/move-article-command.js [hunk 1]
```diff
@@ -71,7 +71,8 @@ export default class MoveArticleCommand {
       articleContainerElement.getMaxOffset()
     );
     const articleHtml = articles.reduce(
-      (html, article) => (html += article.boundNode.outerHTML),
+      (html, article, index) =>
+        (html += this.generateArticleHtml(article, index)),
```
Comment (nvdk) [inline addon/commands/move-article-command.js:75]: couldn't we leverage the insert-article command here and limit the logic needed to generate the html?

Comment (lagartoverde) [reply addon/commands/move-article-command.js:75]: The problem is that the insert article command as it name says inserts the article, and I would like to keep everything in one insertion for performance reasons

Comment (lagartoverde) [reply addon/commands/move-article-command.js:75]: I could export the generateArticleHtml function to a common file, but I think they are different enough now for it to not be worth it

Comment (nvdk) [reply addon/commands/move-article-command.js:75]: fair enough, might make sense to include that in a comment

Comment (lagartoverde) [reply addon/commands/move-article-command.js:75]: The part of one insertion per operation or the exporting the function one?

**Review by nvdk — APPROVED** [review]

**Pull request merged**